QUICKSTART-ECS.md

```diff
@@ -36,38 +36,63 @@ aws ecs --region us-west-2 create-cluster --cluster-name bottlerocket
 
 ## Finding an AMI
 
-You can either use an official AMI provided by Amazon or build an AMI yourself using
-[our guide](https://github.com/bottlerocket-os/bottlerocket/blob/develop/BUILDING.md).
+You can either build an AMI yourself using [our guide](BUILDING.md), or use an official AMI provided by Amazon.
```

Comment:
For a quickstart, I think pointing to the official AMIs first ends up being a better experience.  I prefer the language that was already here (and maybe we make the EKS one match).

Reply:
I'm thinking it might be best to remove the BUILDING link here altogether.  This is a quickstart for people to launch Bottlerocket.  The README has links to BUILDING right after the links to the quickstarts for people who have more advanced uses.

Reply:
That works for me too.

Reply:
Added a commit with that simplification.